File: prop-core/src/main/java/io/pleo/prop/core/PropException.java
package io.pleo.prop.core;

public class PropException extends RuntimeException {
  public PropException(String message) {
    super(message);
  }

  public PropException(String message, Throwable cause) {
    super(message, cause);
  }

  public PropException(Throwable cause) {
    super(cause);
  }
}

File: prop-core/src/main/kotlin/io/pleo/prop/core/PropException.kt
package io.pleo.prop.core

open class PropException(
    message: String? = null,
    cause: Throwable? = null,
) : RuntimeException(message, cause) {
    constructor(message: String?) : this(message, null)
    constructor(cause: Throwable?) : this(null, cause)
}
